usersession/agent/rest_api.go

```diff
@@ -220,6 +291,53 @@ func postServiceControl(c *Command, r *http.Request) Response {
 	return impl(&inst, sysd)
 }
 
+func unitStatusToClientUnitStatus(units []*systemd.UnitStatus) []*client.UserServiceUnitStatus {
```

Comment:
Why not add json tags to the sytemd definition and avoid two definitions and translation?

Reply:
It's a good question, it was to have a stable interface between the daemon and client instead of relying on the systemd structure in this case. It felt a bit too decoupled that the systemd interface would suddenly have serializeable requirement without being directly tied to any interface. I'll happily change it if @pedronis agrees as it simplifies things.